### ISSUE TITLE
i can't add special price to configurable products magneto 2

### ISSUE THREAD

**Issue by mazenelmasry** [magento/magento2#16360]
### Preconditions

1. Magento CE 2.2.2 with Sample Data
2. PHP 7.0

### Steps to reproduce

1. Create configurable product
2. Make configurable product with multi size but same price

### Expected result

1. special Price for configurable product should be displayed

### Actual result

1. By default in Magento 1 shows the price before the reduction and the original price in all sizes offered but in Magento 2 I can not add a discounted price to appear on the front

special price not showing configurable products magento 2

**Comment by magento-engcom-team** [comment]
Hi @mazenelmasry. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@mazenelmasry do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by ananth-iyer** [comment]
@mazenelmasry Hi, 
We also found that we can't added price in Configuration product. So we gone through Magento 2 code base and found that any child product with lowest price will be default price for configurable product.

So then we added special price to child product and it shows in configurable product. Because configurable products are dependent on other child products for price, inventory management, etc.

Yes in Magento 1 we can add special price in Configurable product but something needs to be different with each other.
